timeservice.go

```diff
@@ -276,7 +281,10 @@ func runRelay(configFile, daemonAddr string, localAddr *snet.UDPAddr) {
 		log.Fatal("unexpected configuration", zap.Int("number of peers", len(netClocks)))
 	}
 
-	server.StartIPServer(ctx, log, snet.CopyUDPAddr(localAddr.Host))
+	provider := &server.Provider{}
+	provider.Init(1)
+
```

Comment:
Need to start StartNTSKEServer here as well, right?

Reply:
Done